### ISSUE TITLE
figshare adapter exception when creating checksum

### ISSUE THREAD

**Issue by datadavev** [DataONEorg/SlenderNodes#19]
```
Traceback (most recent call last):
  File "oai-pmh_adapter_figshare.py", line 370, in <module>
    main()
  File "oai-pmh_adapter_figshare.py", line 44, in main
    harvest_oai_pmh(mn_config_dict)
  File "oai-pmh_adapter_figshare.py", line 89, in harvest_oai_pmh
    process_record(record_el, node_dict, counter_dict, gmn_client)
  File "oai-pmh_adapter_figshare.py", line 123, in process_record
    gmn_client.create_or_update_if_new(pid, sid, record_datetime, scimeta_pyxb, counter_dict)
  File "oai-pmh_adapter_figshare.py", line 284, in create_or_update_if_new
    self.create_new_sciobj(scimeta, pid, sid, record_datetime)
  File "oai-pmh_adapter_figshare.py", line 321, in create_new_sciobj
    scimeta_str, pid, sid, record_datetime
  File "oai-pmh_adapter_figshare.py", line 343, in _generate_sysmeta
    sysmeta_pyxb.checksum = d1_common.checksum.create_checksum_object_from_string(
AttributeError: module 'd1_common.checksum' has no attribute 'create_checksum_object_from_string'
```

**Comment by datadavev** [comment]
Looks like the d1_common api changed. Try `create_checksum_object_from_bytes`

**Comment by datadavev** [comment]
fixed in Bugs/figshare #21
